agent/src/softwarecontaineragent.cpp

```diff
@@ -18,6 +18,7 @@
  */
 
 #include <cstdint>
+#include <lxc/lxccontainer.h>
```

Comment:
Could you add so that we link against LXC in the CMakeLists.txt for the agent as well?